power_grid_model_c/power_grid_model/include/power_grid_model/common/statistics.hpp

```diff
@@ -0,0 +1,181 @@
+// SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#pragma once
+
+#include "common.hpp"
+#include "three_phase_tensor.hpp"
+
+/**
+ * @file statistics.hpp
+ * @brief This file contains various structures and functions for handling statistical representations of
+ * randomly distributed variables(RDV) used in the Power Grid Model, like in the State Estimation algorithms to
```

Comment:
`randomly distributed variable` is quite a strange expression. For this specific context, 'random variable' is accurate and comcise enough. 
```suggestion
 * random variables (RandVar) used in the Power Grid Model, like in the State Estimation algorithms to
```
or
```suggestion
 * random variables (RV) used in the Power Grid Model, like in the State Estimation algorithms to
```